cluster/environments/azure-single-keyvault/main.tf

```diff
@@ -1,6 +1,6 @@
-terraform {
+/*terraform {
```

Comment:
As above - let's verify if backend should be tested / enabled or not.  There is not a way to selectively disable it.